ftw/publisher/receiver/tests/assets/basic_folder_dx.json

```diff
@@ -1,5 +1,6 @@
 {
     "unicode:backreferences_adapter": {},
+    "unicode:constrain_types_adapter": {},
```

Comment:
What does this line do? Can you maybe point me to a resource 😇

Reply:
It's a consequence of 4teamwork/ftw.publisher.core#66
this file is test input for test_unserialize_fields